snippets/computed.cson

```diff
@@ -0,0 +1,146 @@
+'.source.coffee':
+
+
+
+
+  'Ember Snippets: computed properties':
+    'prefix': 'computed--help'
+    'body':
+      """
+      ${1:
```

Comment:
I find it unnecessary to have help. The way i see that, snippets shouldn't have that. It should just work by knowledge you already have in mind. If you have to look it up, it will waste time